libbeat/outputs/elasticsearch/api_integration_test.go

```diff
@@ -105,8 +104,8 @@ func TestIngest(t *testing.T) {
 	}
 
 	client := getTestingElasticsearch(t)
-	if strings.HasPrefix(client.Connection.version, "2.") {
-		t.Skip("Skipping tests as pipeline not available in 2.x releases")
+	if client.Connection.version.Major < 5 {
```

Comment:
Hopefully we can clean up this code one day as we should not test anymore against 2.x I think in the future.